### DISCUSSION TITLE
Updates about ``conan search`` endpoint spped

### DISCUSSION THREAD

**Discussion by memsharded** [renovatebot/renovate#22071] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

Hi there!

In https://docs.renovatebot.com/modules/manager/conan it still reads a warning:

> The Conan package manager is disabled by default due to slowness in the Conan API. We recommend you only enable it for low volume experimental purposes until [issue #14170](https://github.com/renovatebot/renovate/issues/14170) is resolved.

That links to closed issue https://github.com/renovatebot/renovate/issues/14170

The slowness of the Conan server HTTP API was finally addressed by obtaining versions from the Github https://github.com/conan-io/conan-center-index repository. 

Also, the slowness problem of the API was addressed, and now ``conan search`` commands should be much faster. So maybe the above comment could be revisited:

- The implementation could stick to query the Github repo, and extract versions from there. I think this is a good approach, fast, cheap, and easy, as all the version information there is structured in yaml files, easy to extract. I don't see a downside for this for ConanCenter-like repos, I guess the reason could be the desired to scan user private repos too?
- The implementation could switch to doing ``conan search`` API calls to the server. It will probably require a bit less work, as it won't be necessary to parse the yaml files, and it can also target private repositories. I'd try in any case to not hammer the server with thousands of queries, even if it has been optimized and it is much faster, it would be better to not put excessive load.

**Comment by rarkins** [comment]
Hi, I'm happy for us or anyone to remove the warning and re-enable Conan by default. I think we just forgot to at the time. We could make this change in a non-major release like a regular "new feature" as it's not what I'd consider breaking.
